Rules/UseConsistentIndentation.cs

```diff
@@ -195,15 +195,20 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                             {
                                 ++tempIndentationLevel;
                             }
-                            else
+
+                            // check for comments in between multi-line commands with line continuation
+                            if (k > 2 && tokens[k - 1].Kind == TokenKind.NewLine
```

Comment:
Given the way `k` is not very local, I think it could do with a name change. This switch statement feels pretty big to be embedded directly in a loop like this

Reply:
Ok, I've renamed it to `tokenIndex`